packages/keyboard-shortcuts/src/store/reducer.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Internal dependencies
+ */
+import type { ShortcutAction, WPShortcutKeyCombination } from './actions';
+
+interface ShortcutState {
+	category: string;
+	keyCombination: WPShortcutKeyCombination;
+	aliases?: WPShortcutKeyCombination[];
+	description: string;
+}
+
+type ShortcutsState = Record< string, ShortcutState >;
```

Comment:
Can we please move these types to a `types.ts` file? That means we will have less changes to the file and its git history will be preserved.